terraform/events.tf

```diff
@@ -0,0 +1,27 @@
+# This rule triggers the healthcheck lambda every minute
+resource "aws_cloudwatch_event_rule" "trigger_healthcheck" {
+    name                = "${local.domain_name_base[var.environment]}-healthcheck-trigger"
+    description         = "Fires the healthcheck lambda function every minute"
+    schedule_expression = "rate(1 minute)"
+}
+
+# Targets the healthcheck lambda
+resource "aws_cloudwatch_event_target" "sml_site_trigger_healthcheck" {
+    rule      = "${aws_cloudwatch_event_rule.trigger_healthcheck.name}"
+    target_id = "check_sml_site"
+    arn       = "${aws_lambda_function.healthcheck.arn}"
+    input     = jsonencode({
+                  "site"            : "https://${local.domain_name_base[var.environment]}",
```

Comment:
This should be URL and we should have a tf variable of that is declared using the environment.

Reply:
Done